worlds/kth_space_lab.sdf

```diff
@@ -0,0 +1,335 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<sdf version="1.9">
+  <world name="kth_space_lab">
```

Comment:
Small nitpik: Can we remove the underscores? This would make the make target quite complicated.
Maybe `kthspacelab`?